### DISCUSSION TITLE
Action for running any action

### DISCUSSION THREAD

**Discussion by dipakc** [TriliumNext/Trilium#1594] [Ideas]
It would nice to have an action for running any action by typing its name. Is this already there? This will be useful especially on Mac, I will just use a keyboard shortcut for the "Run action" action and type in the action I want to run. No need to set (and remember) keyboard shortcuts for all the actions.

**Comment by zadam** [comment]
Hi, this is sometimes called "command palette" but unfortunately this is not available in Trilium.

However I agree that this would be useful and it makes sense to add it to the backlog ...
